chromium_src/chrome/browser/ui/views/download/download_ui_context_menu_view.cc

```diff
@@ -32,26 +32,54 @@ ui::SimpleMenuModel* DownloadUiContextMenuView::GetMenuModel() {
     return model;
   }
 
+  MaybeAddCopyDownloadLinkMenuItem(download, model);
```

Comment:
why are we overriding this in `DownloadUiContextMenuView` when upstream adds them in `DownloadUiContextMenu`?

Reply:
Also uploaded draft for this one here: https://github.com/brave/brave-core/pull/32243